ruby/neo4j/driver/internal/packstream/pack_stream.rb

```diff
@@ -0,0 +1,454 @@
+module Neo4j::Driver
+  module Internal
+    module Packstream
+      class PackStream
+        TINY_STRING = 0x80
+        TINY_LIST = 0x90
+        TINY_MAP = 0xA0
+        TINY_STRUCT = 0xB0
+        NULL = 0xC0
+        FLOAT_64 = 0xC1
+        FALSE = 0xC2
+        TRUE = 0xC3
+        RESERVED_C4 = 0xC4
+        RESERVED_C5 = 0xC5
+        RESERVED_C6 = 0xC6
+        RESERVED_C7 = 0xC7
+        INT_8 = 0xC8
+        INT_16 = 0xC9
+        INT_32 = 0xCA
+        INT_64 = 0xCB
+        BYTES_8 = 0xCC
+        BYTES_16 = 0xCD
+        BYTES_32 = 0xCE
+        RESERVED_CF = 0xCF
+        STRING_8 = 0xD0
+        STRING_16 = 0xD1
+        STRING_32 = 0xD2
+        RESERVED_D3 = 0xD3
+        LIST_8 = 0xD4
+        LIST_16 = 0xD5
+        LIST_32 = 0xD6
+        RESERVED_D7 = 0xD7
+        MAP_8 = 0xD8
+        MAP_16 = 0xD9
+        MAP_32 = 0xDA
+        RESERVED_DB = 0xDB
+        STRUCT_8 = 0xDC
+        STRUCT_16 = 0xDD
+        RESERVED_DE = 0xDE
+        RESERVED_DF = 0xDF
+        RESERVED_E0 = 0xE0
+        RESERVED_E1 = 0xE1
+        RESERVED_E2 = 0xE2
+        RESERVED_E3 = 0xE3
+        RESERVED_E4 = 0xE4
+        RESERVED_E5 = 0xE5
+        RESERVED_E6 = 0xE6
+        RESERVED_E7 = 0xE7
+        RESERVED_E8 = 0xE8
+        RESERVED_E9 = 0xE9
+        RESERVED_EA = 0xEA
+        RESERVED_EB = 0xEB
+        RESERVED_EC = 0xEC
+        RESERVED_ED = 0xED
+        RESERVED_EE = 0xEE
+        RESERVED_EF = 0xEF
+
+        PLUS_2_TO_THE_31  = 2147483648
```

Comment:
power mathematic notations can be written with double splat `**` in Ruby.
e.g. for above case `2**31`
I find that more readable, But since the constant name itself conveys purpose, I don't have hard opinion on changing this.